ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage  extends Page {
+    
+    private  HomePage homePage;
+    private static String name = "john";
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By PERSON_NAME = By.cssSelector("#relationship > p:nth-child(2) > input.person-typeahead.ng-pristine.ng-valid.ng-empty.ng-touched");
```

Comment:
If you get a better element reference for `PERSON_NAME` would be a good deal. One challenge with locating a page element through `nth-child(x)` value is that things get broken when UI changes comes in and element can not be located any more via `x-` value!

Reply:
i would go with this cssSelector element , Alternatively using xpath

Reply:
@sherrif10 did you see the concern on current `cssSelector` value ? I think we need to put code sustainability in mind to reduce the maintenance burden. If we would go with the current set value this means time will come when that value will break the test due to the reason I gave above this is another pain looking through the test locating where things are breaking.